tests/test_a11y_nodes.py

```diff
@@ -0,0 +1,90 @@
+import json
+import subprocess
+import unittest
+from pathlib import Path
+from unittest import mock
+
+from adt_press.nodes import a11y_nodes
+
+
+class A11yNodesTests(unittest.TestCase):
```

Comment:
Is this test adding real coverage or appreciable testing? I'm still wrapping my head around this new AI world of ours but one thing I'm already noticing is there is a real cost to having a zillion AI written tests in that they tend to be fragile and testing things that don't really matter and doubly nobody really understands why they are testing what they are testing. So you end up wasting a lot of time trying to fix them.